### ISSUE TITLE
^D not working on Windows

### ISSUE THREAD

**Issue by linusg** [alimpfard/zigline#10]
![image](https://github.com/alimpfard/zigline/assets/19366641/7579f6e1-020e-47cb-9fa7-ee08594bafc5)

(the `error: EndOfStream` appears on ^C, not the preceding ^D)

**Comment by alimpfard** [comment]
:thinking: I think the EOF sequence in windows is ^Z, mind testing that?

**Comment by alimpfard** [comment]
Yeah that's expected, we're going to have to move out of the NonInteractive mode to actually receive unbuffered keys, which is a bit of a chore as windows doesn't have a termios interface, and zig doesn't have bindings for the console interface.
